requirements.txt

```diff
@@ -2,5 +2,7 @@ numpy
 scipy
 scikit-image
 boto
+pillow
```

Comment:
OK, my environment must have been borked. Just went back and tried it with a fresh environment and you're right -- `pillow` is included by `scikit-image` so that's all good. Incidentally, I also discovered that `six` also gets pulled in by `scikit-image`, so I'll remove that from the dependency list as well.